integration/samples.sh

```diff
@@ -1,37 +1,63 @@
-#!/bin/bash
-set -e
+#!/bin/sh
+# NOTE: When making changes, make sure to run the script through ShellCheck!
+# @see https://github.com/koalaman/shellcheck
 
-FIND_COMMAND=""
+# Exit script on error.
+set -o errexit
+# Disable pathname expansion.
+set -o noglob
+# Error on unset (non-special) parameter expansion.
+set -o nounset
 
-if [[ $OSTYPE == "win32" ]]; then
-    FIND_COMMAND="dir /b /o:n /ad"
+DIR_NAME_LIST=""
+if [ "${OSTYPE:-}" = "win32" ]; then
+    DIR_NAME_LIST=$(dir /b /o:n /ad "integration/samples/")
 else
-    FIND_COMMAND="find ./integration/samples/* -maxdepth 0 -type d"
+    DIR_NAME_LIST=$(dir --format=vertical --escape "integration/samples/")
```

Comment:
I am having some issues with this locally. On OSX, dir doesn't exist.

Reply:
Checkout the new PR where I've switched the integrations build script to JS: https://github.com/ng-packagr/ng-packagr/pull/1853